src/config/enterprise.rs

```diff
@@ -135,7 +135,7 @@ struct PipelinesAuth<'a> {
 
 /// Holds the relevant fields for reporting a configuration to Datadog Observability Pipelines.
 struct PipelinesStrFields<'a> {
-    config_version: &'a str,
+    config_version_hash: &'a str,
```

Comment:
Should this also be the expanded `configuration_version_hash`?

It may seem a bit redundant, but these internal discrepancies can be confusing when multiplied, especially when considering the interface between Vector and the API.